lib/dealership.rb

```diff
@@ -0,0 +1,44 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :inventory_count
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(car)
+    @inventory << car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    if @inventory_count == 0
+      false
+    else
+      true
+    end
+  end
+
+  def cars_by_make(make)
+    @inventory.select do |car|
+      car.make == make
+    end
+  end
+
+  def total_value
+    @total = []
```

Comment:
You would not put the `@` sign before your accumulator. You would just write
```
total = [ ]
```
Nice use of `.each` here. Is there an enumerable that sums? 😉